iOS Example/iOS Example/ViewController.swift

```diff
@@ -56,8 +56,38 @@ class ViewController: UIViewController {
 		tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorButtonText) { self.bounceButton.setTitleColor($0, for: .normal) })
 		tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorBackground) { self.view.backgroundColor = $0 })
 		tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorText1) { self.titleLabel.textColor = $0 })
-		tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorText2) { self.bodyLabel.textColor = $0 })
-
+        tweakBindings.insert(ExampleTweaks.bind(ExampleTweaks.colorText2) { self.bodyLabel.textColor = $0 })
+        
+        // If you used `TweaksCallbacks` type, you can subscribe to those tweaks with callbacks.
+        // Those will be run in order.
```

Comment:
So there I personally haven't found a real use where the order would be crucial, but I believe that if we can preserve that order, why not do it - maybe it will be useful to someone.

I think I understand that the order might not be super obvious, but then again, using `bind` wouldn't really solve that problem.

And the way I see this feature is that it's not very likely that callbacks to one tweak would be in all three spaces as in the example - ideally, you should have one source of truth and other components should react to change in that source. 
Like when you want to have a button to force log out or invalidate token, you shouldn't have to change that state via tweaks in several different places, because that means there's probably something wrong with your code.